src/core/Akka/Actor/SupervisorStrategy.cs

```diff
@@ -350,7 +350,8 @@ public OneForOneStrategy(int maxNrOfRetries, int withinTimeMilliseconds, IDecide
         /// Constructor that accepts only a decider and uses reasonable defaults for the other settings
         /// </summary>
         /// <param name="localOnlyDecider">TBD</param>
-        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider) : this(-1, -1, localOnlyDecider, true)
+        /// <param name="loggingEnabled">TBD</param>
+        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider, bool loggingEnabled = true) : this(-1, -1, localOnlyDecider, loggingEnabled)
```

Comment:
Here I break the public API. Contract wise I kind of didn't break it - I just propagated the loggingEnabled flag and set the default value to true, so that not to break the previous usages of this constructor. What else I could do here:
1. I could add a separate constructor but this would break the api check as well
2. I could you one of the existing constructors where loggingEnabled flag is available for setting, but the Constructor I changed is the easiest to use...